src/services/selfHosted/useSelfHostedUserList.spec.tsx

```diff
@@ -192,4 +193,25 @@ describe('useSelfHostedUserList', () => {
       })
     })
   })
+
+  describe('endpoint returns invalid data', () => {
+    beforeEach(() => {
```

Comment:
jw, do we need this console.error() thing?

Reply:
We throw it in for these invalid responses because it will log out that there was an error which is expected, and we don't want to clutter the console output with false positives.

Reply:
Got it, thanks for clarifying that